test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -283,6 +283,101 @@ plt.show()`;
 
 				await app.workbench.positronPlots.waitForNoPlots();
 			});
+
+			it('Python - Verifies bqplot Python widget [C720869]', async function () {
+				const app = this.app as Application;
+
+				const script = `import bqplot.pyplot as bplt
+import numpy as np
+
+x = np.linspace(-10, 10, 100)
+y = np.sin(x)
+axes_opts = {"x": {"label": "X"}, "y": {"label": "Y"}}
+
+fig = bplt.figure(title="Line Chart")
+line = bplt.plot(
+	x=x, y=y, axes_options=axes_opts
+)
+
+bplt.show()`;
+
+				await app.workbench.positronConsole.pasteCodeToConsole(script);
+				await app.workbench.positronConsole.sendEnterKey();
+				await app.workbench.positronPlots.waitForWebviewPlot('.svg-figure');
+
+				await app.workbench.positronPlots.clearPlots();
+
```

Comment:
Not waiting for plots to be gone because structure used to identify plots remains.